staging/kos/pkg/controllers/ipam/ipam.go

```diff
@@ -905,26 +897,13 @@ func (x ParsedLock) Equal(y ParsedLock) bool {
 
 func (x ParsedLock) IsBetterThan(y ParsedLock) bool {
 	if x.CreationTime != y.CreationTime {
-		return x.CreationTime.Before(y.CreationTime)
+		return x.CreationTime.After(y.CreationTime)
```

Comment:
Why reverse this choice?  Preferring the earlier one was a deliberate choice, for the sake of stability.

Reply:
For consistency with what is stated in the comments where `Best` (which uses `IsBetterThan`) is used: https://github.com/MikeSpreitzer/kube-examples/blob/make-subnets-validation-correct/staging/kos/pkg/controllers/ipam/ipam.go#L582-L585. Plus, the explanation in said comments seemed reasonable, therefore I changed `IsBetterThan` rather than the comment.

Reply:
Oh dang, I forgot to fix that comment.  That reflects older thinking.  I first thought as outlined in that comment, but later decided that stability is the more important goal, and switched to the code you see.  So I recommend fixing the comment and keeping `Before` in `IsBetterThan`.

Reply:
I reverted the change and updated the comment in 8dd139c. But, for the sake of my understanding, what do you mean by stability, and why is the oldest best towards stability?

Reply:
Stability is not thrashing the choice of address.  Once one has been allocated, prefer that one over addresses allocated later.  I saw situations where there was always a new address picked before all the notifications came in for the previously picked address.